packages/universal-components/src/FormLabel/FormLabel.js

```diff
@@ -13,7 +13,8 @@ type Props = {|
   +disabled?: boolean,
   +required?: boolean,
   +inlineLabel?: boolean,
-  +style?: StylePropType,
+  +labelContainerStyle?: StylePropType,
```

Comment:
What about rename `labelContainerStyle` to `containerStyle`? From my perspective it's more clear this way and it's some kind of duplicity of the component name (`label`).

`labelTextStyle` could be also renamed in the same style to `textStyle`

What do you think?

Reply:
I agree with the suggestions!

Reply:
Great! changed it.